jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What is this file for?

Reply:
It is for tooling: Vetur (https://vuejs.github.io/vetur/guide/setup.html)